### ISSUE TITLE
Chatting during a game shouldn't make the game pause

### ISSUE THREAD

**Issue by soundmud** [soundmud/soundrts#59]
The interface's focus should go into a mode allowing the game to run in parallel.

**Comment by soundmud** [comment]
Actually, there is another issue: when the chat message is spoken on the receptor, the client is pausing, so with a message of more than 20 seconds you could possibly disconnect your opponent.
 
@thgcode You might have noticed it, since you have changed some blocking voice.menu() into non-blocking voice.item().
